.gitmodules

```diff
@@ -1,3 +1,7 @@
+[submodule "mbed-os"]
+    path = mbed-os
+    url = https://github.com/ARMmbed/mbed-os.git
+
 [submodule "uwrt-mars-rover-hw-bridge"]
-	path = uwrt-mars-rover-hw-bridge
-	url = https://github.com/uwrobotics/uwrt_mars_rover_hw_bridge.git
+    path = uwrt-mars-rover-hw-bridge
+    url = ../uwrt_mars_rover_hw_bridge.git
```

Comment:
could you explain the reason for this change?

Reply:
it make it relative to the main repo. so if firmware was closed using ssh, then bridge will match that.